README.md

```diff
@@ -37,6 +37,8 @@ This repository contains algorithms and data structures implemented in C# for ed
 		* [Recursive Binary](./Algorithms/Search/RecursiveBinarySearcher.cs)
 		* [Linear](./Algorithms/Search/LinearSearcher.cs)
 		* [FastSearch](./Algorithms/Search/FastSearcher.cs)
+		* [Substring](./Algorithms/Search/Substring)
+			* [Rabin Karp](./Algorithms/Search/Substring/RabinKarp.cs)
```

Comment:
I believe this should be in the String section next to [Naive String Search]

Reply:
Then we should have a look at #159 :D